plugins/processors/ec2tagger/ec2tagger.go

```diff
@@ -118,20 +59,17 @@ type Tagger struct {
 	Filename  string `toml:"shared_credential_file"`
 	Token     string `toml:"token"`
 
-	ec2TagCache    map[string]string
-	instanceId     string
-	imageId        string // aka AMI
-	instanceType   string
-	started        bool
-	region         string
-	ec2Provider    ec2ProviderType
-	ec2            ec2iface.EC2API
-	ec2metadata    ec2Metadata
-	refreshTicker  *time.Ticker
-	shutdownC      chan bool
-	tagFilters     []*ec2.Filter
-	metadataLookup metadataLookup
-	ebsVolume      *EbsVolume
+	ec2TagCache         map[string]string
+	started             bool
+	ec2Provider         ec2ProviderType
+	ec2API              ec2iface.EC2API
+	ec2MetadataProvider ec2MetadataProviderType
+	ec2MetadataRespond  ec2MetadataRespond
```

Comment:
Nit - It's improper to have the name of the variable shadow the name of the type.